packages/node/src/lib/ReadMe.ts

```diff
@@ -55,160 +55,165 @@ interface ReadMeVersion {
   version: string;
 }
 
-// See comment at the auth definition below
-let readmeAPIKey = '';
-let readmeProjectData: GetProjectResponse200 | undefined;
-let readmeVersionData: ReadMeVersion[] | undefined;
```

Comment:
moving everything into a class will also contain this data to the specific instance. as-is it'd be difficult/impossible to spin up different instances of the SDK against different projects (like a group of microservices) because this data would get clobbered between each